<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lb_instrument">
        <property name="text">
         <string>Instrument:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_instrument">
        <item>
         <property name="text">
          <string>IRIS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OSIRIS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TOSCA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VESUVIO</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_analyser">
        <property name="text">
         <string>Analyser:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_analyser">
        <item>
         <property name="text">
          <string>graphite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fmica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_reflection">
        <property name="text">
         <string>Reflection:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_reflection">
        <item>
         <property name="text">
          <string>002</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>004</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_input">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lb_groupingMethod">
        <property name="text">
         <string>Grouping Method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_calibration"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_input">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>sample</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_sampleLogName">
        <property name="text">
         <string>Sample Log Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_calibration">
        <property name="text">
         <string>Calibration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cb_groupingMethod">
        <item>
         <property name="text">
          <string>Individual</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_browseCalibration">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_browseInput">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tb_quickrun">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="cb_plotOptions">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_plot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pb_help">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open interface help page in your web browser.</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="pb_pythonExport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Export a Python script for the algorithms run on this tab.</string>
        </property>
        <property name="text">
         <string>Py</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="pb_run">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Run conversion to energy process.</string>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QPushButton" name="pb_manageDirectories">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Manage Directories</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
